packaging/test_irods_client_rest_cpp.py

```diff
@@ -932,3 +932,73 @@ def test_trim_collection(self):
                 admin.run_icommand(['irm', '-r', '-f', coll_name])
                 for resc in resources:
                    admin.run_icommand(['iadmin', 'rmresc', resc])
+
+
+def collection_exists(session, full_collection_path):
+    out = session.run_icommand(['iquest',
```

Comment:
`run_icommand` returns a tuple?

Reply:
Yeah, notice the `[0]` at the end of the line (well, I guess it's on the next line, 939).

Reply:
oh, yeah, totally saw that.